src/libraries/System.Text.Json/tests/Serialization/ContinuationTests.cs

```diff
@@ -10,502 +11,324 @@
 
 namespace System.Text.Json.Serialization.Tests
 {
-    public static class ContinuationTests
+    public static partial class ContinuationTests
     {
-        // To hit all possible continuation positions inside the tested object,
-        // the outer-class padding needs to be between 5 and 116 bytes long.
-
-        // {"S":"xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx","C":{"A":null,"B":"Hello","C":42,"D":null,"E":3.14E+17,"F":null,"G":true,"H":null,"I":[42,17],"J":{"A":null,"B":7}}}
-        // |<------------------------------------------------128 byte buffer------------------------------------------------------------->|
-        // {"S":"xxxxx","C":{"A":"Hello","B":null,"C":42,"D":null,"E":3.14E+17,"F":null,"G":true,"H":null,"I":[42,17],"J":{"A":null,"B":7}}}
-
-        private const int MinPaddingLength = 5;
-        private const int MaxPaddingLength = 116;
+        private static readonly Func<string, string>[] s_payloadTweaks = new Func<string, string>[]
+        {
+            payload => payload,
+            payload => payload.Replace("null", "nullX"),
+            payload => payload.Replace("e+17", "e+-17"),
```

Comment:
Would testing the following be worthwhile? To also check for `JsonException` for booleans in continuation cases

```cs
payload => payload.Replace("true", "trueX"),
payload => payload.Replace("true", "falseX"),
```

Reply:
We should be careful about adding too many permutations of invalid tests, particularly here. There are diminishing returns, and we have such tests in a lot of other places already, where they are more impactful.